third_party/move/move-binary-format/src/file_format.rs

```diff
@@ -3365,22 +3365,39 @@ pub fn empty_module_with_dependencies_and_friends<'a>(
     dependencies: impl IntoIterator<Item = &'a str>,
     friends: impl IntoIterator<Item = &'a str>,
 ) -> CompiledModule {
-    // Rename this empty module.
+    empty_module_with_dependencies_and_friends_at_addr(
+        AccountAddress::ZERO,
+        module_name,
+        dependencies,
+        friends,
+    )
+}
+
+/// Creates an empty compiled module with specified dependencies and friends. All
+/// modules (including itself) are stored at the specified address.
+pub fn empty_module_with_dependencies_and_friends_at_addr<'a>(
+    address: AccountAddress,
+    module_name: &'a str,
+    dependencies: impl IntoIterator<Item = &'a str>,
+    friends: impl IntoIterator<Item = &'a str>,
+) -> CompiledModule {
+    // Rename this empty module and rest its address.
```

Comment:
I know it's an old comment, but nits:
is 'rest' supposed to be 'test'?
if this is a TODO let's add a TODO: 
also what does renaming it mean?

Reply:
Meant "reset", also explanation comment, maybe not needed?